contrib/kubespray/quick-start/services-configuration.yaml.template

```diff
@@ -34,6 +34,28 @@ cluster:
     # Must be lower case, e.g., regsecret.
     secret-name: pai-secret
 
+{% if env["cfg"]["enable_docker_cache"]|default(false) is sameas true %}
+docker-cache:
+  enabled: {{ env["cfg"]["enable_docker_cache"] }}
+  {% if "docker_cache_storage_backend" in env["cfg"] %}
+  storage_backend: {{ env["cfg"]["docker_cache_storage_backend"]}}
+    {% if env["cfg"]["docker_cache_storage_backend"] == "azure" %}
+  azure_account_name: {{ env["cfg"]["docker_cache_azure_account_name"] }}
+  azure_account_key: {{ env["cfg"]["docker_cache_azure_account_key"] }}
+  azure_container_name: {{ env["cfg"]["docker_cache_azure_container_name"] }}
+    {% elif env["cfg"]["docker_cache_storage_backend"] == "filesystem" %}
+  fs_mount_path: {{ env["cfg"]["docker_cache_fs_mount_path"]}}
+    {% else %}
+    {% endif %}
+  {% endif %}
+  {% if "docker_cache_remote_url" in env["cfg"] %}
+  remote_url: {{ env["cfg"]["docker_cache_remote_url"] }}
+  {% endif %}
+  {% if "docker_cache_registry_htpasswd" in env["cfg"] %}
+  registry-htpasswd: {{ env["cfg"]["docker_cache_registry_htpasswd"] }}
+  {% endif %}
+{% else %}
+{% endif %}
```

Comment:
Please pay attention to whitespace control in jinja2: https://jinja.palletsprojects.com/en/2.11.x/templates/#whitespace-control